### PR TITLE
Resolve issue 8347 by checking for nullptr.  

### PR DESCRIPTION
**Pull request by wrdouglass**
Additionally resolve a couple other similar issues for keyboard accelerators in inputs & mixers

### CODE REVIEW

**Comment by elecpower** [comment]
Fixes #8347 (link to Issue)

File: companion/src/modeledit/inputs.cpp [hunk 1]
```diff
@@ -351,6 +351,8 @@ void InputsPanel::expoOpen(QListWidgetItem *item)
   if (!item)
     item = ExposlistWidget->currentItem();
 
+  if (item == nullptr) return;
```
Comment (elecpower) [inline companion/src/modeledit/inputs.cpp]: @wrdouglass thanks for the PR we appreciate the assistance. Could you please change this and similar changes to use the project's current standards of line 351 for nullptr test and if statement(s) on following lines to test.

Comment (wrdouglass) [reply companion/src/modeledit/inputs.cpp]: i'm not quite sure what you are asking

do you mean

if (item == nullptr)
  return;
  
 i wondered about that but i could see that there were varying standards in that file itself and went with what i was accustomed to, however i can certainly change that.

**Comment by 3djc** [comment]
This might come handy : https://github.com/opentx/opentx/wiki/OpenTX-Code-Style-Guide

And yes, code reformatting is still in progress :)

**Pull request merged**